packages/alpinejs/src/directives/x-for.js

```diff
@@ -135,6 +136,12 @@ function loop(el, iteratorNames, evaluateItems, evaluateKey) {
         for (let i = 0; i < removes.length; i++) {
             let key = removes[i]
 
+            //Remove any queued effects that might run after the DOM node has been removed.
+            if (!!lookup[key]._x_effects) {
+                lookup[key]._x_effects.forEach((effect) => {
+                    dequeueJob(effect)
+                })
```

Comment:
Could just call `lookup[key]._x_effects.forEach(dequeueJob)` without the arrow function.